packages/node_modules/@ciscospark/redux-module-activity/src/actions.js

```diff
@@ -285,3 +285,9 @@ export function retryFailedActivity(failedActivity, spark) {
       .catch((error) => dispatch(submitActivityFailure(failedActivity, error)));
   };
 }
+
+export function parseMarkdown(text) {
+  const activityObject = createMessageObject(text);
```

Comment:
Instead of adding this in the actions, lets just export the `createMessageObject` function in index.js:

`packages/node_modules/@ciscospark/redux-module-activity/src/index.js`:
```js
export {createMessageObject} from './helpers';
```